source/ota.c

```diff
@@ -2227,8 +2438,8 @@ static IngestResult_t decodeAndStoreDataBlock( OtaFileContext_t * pFileContext,
                                                const uint8_t * pRawMsg,
                                                uint32_t messageSize,
                                                uint8_t ** pPayload,
-                                               uint32_t * uBlockSize,
-                                               uint32_t * uBlockIndex )
+                                               uint32_t * blockSize,
+                                               uint32_t * blockIndex )
```

Comment:
shall this be `pblockIndex`? Since this is a pointer?

Reply:
Thanks updated in https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/116/commits/3e5b1d96211d212a79fd5aa4a1bd33a51add1e85